perimetre_asa/asaperimetre.py

```diff
@@ -51,6 +51,13 @@ def initGui(self):
         self.dock = AsaPerimetreDock()
         iface.addDockWidget(Qt.RightDockWidgetArea, self.dock)
 
+        action = self.dock.toggleViewAction()
+
+        # Add Perimetre ASA toolbar
+        self.toolbar = iface.addToolBar('&Périmètre asa')
+        self.toolbar.setObjectName("perimetre_asa")
+        self.toolbar.addAction(action)
```

Comment:
Only add the icon to the plugins toolbar.
```suggestion
        iface.addToolBarIcon(action)
```

Reply:
Yes, and we should unload anything loaded I think.

Also fix conflict as stated in https://github.com/3liz/qgis-perimetre_asa-plugin/pull/2#pullrequestreview-599302303